x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/delete.ts

```diff
@@ -18,6 +18,11 @@ export default function createDeleteTests({ getService }: FtrProviderContext) {
 
   describe('delete', () => {
     const objectRemover = new ObjectRemover(supertest);
+    const OtherSpace = Spaces.find(space => space.id === 'other');
```

Comment:
This code repeats in every file.  Honestly not sure if we couldn't just get away with hardcoding the string `other` into all the calls where we're checking cross space access.  Would reduce the code volume a lot here.

Reply:
Fixed in https://github.com/elastic/kibana/pull/45892/commits/d6cbfc84d184e8bca8606d6fe620d610cb6e02a8.

Reply:
Back when the first set of these multi-tests came in, Mike and I talked about all the boilerplate, but the thing was that we didn't know exactly what would make sense in terms of optimizations (eg, sharing code), so anything we could have done would be a pre-optimization.

I think it would be good to track this with a new issue, now that this final bit is in (or at least another good chunk, but I think this was the missing piece).  Hopefully, it will be somewhat obvious, now, to figure out what we can share.

Reply:
But boy is there a lot of duplication in there hahahah!